.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "4.4.2"
+  - '0.10'
+  - 4.4.2
```

Comment:
Ok, cool thanks.

I think it would be good to have `0.10`, `4.4` at the moment. We could look in future to adding the newer node versions as newer node versions are supported on that platform.